release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,447 @@
+
+[id="process-listening-endpoints-rn40"]
+=== Processes listening on endpoints API
+
+{product-title-short} now provides a list of all processes that are listening on ports in secured clusters. This information can help you associate which deployments and associated processes have open ports and better assess cluster security.
+
+This functionality is available for deployments by using the `v1/listening_endpoints` API. Changes to the {product-title-short} web portal for this feature are planned for a future release. In this release, the following known limitations exist:
+
+- It is possible that the PID information attached to an endpoint reflects a process of the same name and parameters, but earlier execution time. Additionally, only the pod ID, process name, process arguments, port, and protocol are guaranteed to be populated for a given deployment.
+- Under rare circumstances, network endpoints reported by the API might have process information reflecting the characteristics of the parent process instead of the process that created the listening endpoint.
+- Only TCP endpoints are currently reported.
+
+For more information, navigate to *Help* -> *API reference* in the {product-title-short} portal.
+
+[id="network-graph-rn40"]
+=== Network graph 2.0 (Technology Preview) updates
+
+:FeatureName: Network graph 2.0
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The Network Graph (2.0 preview) has been updated to address user feedback and add improvements. The following changes have been made:
+
+- Differentiated “filtered” namespaces from “derived” namespaces
+- Changed behavior of the display to dim additional items in the background when an item is selected
+- Anomalous and baseline flows have been separated
+- Added container configurations
+- Renamed “Extraneous flows” to “Inactive flows”
+
+For more information about the Network Graph (2.0 preview), see xref:../operating/manage-network-policies.adoc#network-graph-20-docs[Network graph (2.0 preview)].
+
+[id="fips-compliance-rn40"]
+=== FIPS Compliance
+
+{product-title-short} functionality has been validated on {osp} 4.12 running in Federal Information Processing Standards (FIPS) mode.
+
+[id="token-expiry-alert-rn40"]
+=== Alert messages sent to Central logs for expiring tokens
+
+API tokens are used in {product-title-short} for some system integrations, authentication processes, and system functions. API tokens expire in 1 year. This release adds an automatic notification process to create warning messages for tokens that will expire in less than 1 week. These messages appear in the Central logs. You can configure your system to alert you when you receive these messages so that you know when you need to generate new tokens. For more information, see xref:../configuration/configure-api-token.adoc#about-api-token-expiration_configure-api-token[About API token expiration].
+
+[id="sensor-resync-rn40"]
+=== Improvements for sensor resync (Technology Preview)
```

Comment:
Should "sensor" be initcap'd "Sensor" here? I noticed it was capitalized in the body text.